api/models/linkPreview/findOrCreateByUrl.js

```diff
@@ -1,27 +1,15 @@
 export default function findOrCreateByUrl (url) {
   return LinkPreview.find(url).then(preview => {
-    if (!preview) {
-      return LinkPreview.queue(url)
-      .then(() => {
-        return {status: 'queued'}
-      })
-    }
-
-    if (!preview.get('done')) {
-      return {status: 'loading'}
-    }
-
-    return Object.assign(
-      preview.pick(
-        'id',
-        'url',
-        'image_url',
-        'title',
-        'description',
-        'image_width',
-        'image_height'
-      ),
-      {status: 'loaded'}
+    if (!preview) return LinkPreview.queue(url)
+    if (!preview.get('done')) return
+    return preview.pick(
```

Comment:
We shouldn't call `pick` here, because this is at the model layer. GraphQL will take care of filtering down what's returned to the client.